hack/make-rules/test-e2e-node.sh

```diff
@@ -73,9 +73,9 @@ if [ $remote = true ] ; then
   gubernator=${GUBERNATOR:-"false"}
   image_config_file=${IMAGE_CONFIG_FILE:-""}
   if [[ $hosts == "" && $images == "" && $image_config_file == "" ]]; then
-    image_project=${IMAGE_PROJECT:-"google-containers"}
+    image_project=${IMAGE_PROJECT:-"cos-cloud"}
     gci_image=$(gcloud compute images list --project $image_project \
-    --no-standard-images --regexp="gci-dev.*" --format="table[no-heading](name)")
+    --no-standard-images --regexp="cos-dev.*" --format="table[no-heading](name)")
```

Comment:
I don't know why we use dev, that's why I'm looking for vishh and want to ask him why we should use dev here.
https://github.com/Random-Liu/kubernetes/commit/e9034f1e0aeace5b7a0847101a19397f201257d6#diff-d06d69f47fb79d27f9f55ad0d01be769